### PR TITLE
Pass in Scope to the mgmt credscan step

### PR DESCRIPTION
**Pull request by weshaggard**
mgmt doesn't work against ServiceDirectory it uses Scope so we need to pass that in instead.

### CODE REVIEW

**Comment by JoshLove-msft** [comment]
> Azure Pipelines could not run because the pipeline triggers exclude this branch/path.

This highlights the problem with changing this ci file - it requires manual testing.

**Comment by weshaggard** [comment]
>This highlights the problem with changing this ci file - it requires manual testing.

Yep just added the template file to the set of triggers for the pipeline.

**Review by JoshLove-msft — APPROVED** [review]

File: eng/pipelines/templates/jobs/ci.mgmt.yml [hunk 1]
```diff
@@ -116,4 +116,4 @@ jobs:
       - template: /eng/common/pipelines/templates/steps/credscan.yml
         parameters:
           BaselineFilePath: $(Build.sourcesdirectory)/eng/dotnet.gdnbaselines
-          ServiceDirectory: ${{ parameters.ServiceDirectory }}
\ No newline at end of file
+          ServiceDirectory: ${{ parameters.Scope }}
```
Comment (weshaggard) [inline eng/pipelines/templates/jobs/ci.mgmt.yml:119]:
```suggestion
```

**Pull request merged**